lib/runtime/src/metrics/prometheus_names.rs

```diff
@@ -45,7 +45,10 @@ pub mod frontend_service {
     /// Total number of LLM requests processed
     pub const REQUESTS_TOTAL: &str = "requests_total";
 
-    /// Number of inflight requests
+    /// Number of requests waiting in HTTP queue before receiving the first response.
+    pub const HTTP_QUEUED_REQUESTS: &str = "http_queued_requests";
```

Comment:
can we make this "queued_requests" instead of "http_queued_requests" - not sure the "http" part is adding value here